integration-web/src/main/java/org/symphonyoss/integration/web/resource/WebResourceExceptionHandler.java

```diff
@@ -113,4 +114,11 @@ private ErrorResponse buildErrorResponse(int status, String message) {
     return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(response);
   }
 
+  @ResponseBody
+  @ExceptionHandler(OAuth1Exception.class)
```

Comment:
I believe you could handle AuthorizationException instead of OAuth1.. to make it more generic.